### PR TITLE
Add -tv-elapsed-time

### PR DESCRIPTION
**Pull request by aqjune**
This PR adds `-tv-elapsed-time` that prints the running time of TV.

### CODE REVIEW

**Comment by aqjune** [comment]
Exclude the time taken by LLVM optimizations

**Comment by nunoplopes** [comment]
can you use util/stopwatch.h instead pls?

**Comment by aqjune** [comment]
Hi, can I get this merged? This is useful for estimating the time spent on translation validation only.

**Comment by nunoplopes** [comment]
I'm failing to see the point of this patch. It's measuring the aggregated time of TV across all functions, right?
So it's: process run time - optimization time. Since optimization time is so small when compared to TV time, can't you just use the process run time?
Unless you want to do per function pair, which would make more sense to me. We have a time around vcgen, but not around refinement formula creation nor SMT solving. We miss runaways in those two.

**Comment by aqjune** [comment]
I agree that per function time info should be more useful; updated the patch as suggested
.

File: tv/tv.cpp [hunk 1]
```diff
@@ -142,6 +148,7 @@ optional<smt::smt_initializer> smt_init;
 optional<llvm_util::initializer> llvm_util_init;
 TransformPrintOpts print_opts;
 unordered_map<string, FnInfo> fns;
+unordered_map<string, float> fns_elapsed_time;
```
Comment (aqjune) [inline tv/tv.cpp:151]: Declared a separate map here because if a function `f` has unsupported instructions `fns` may not have a slot for `f`.

**Comment by aqjune** [comment]
The output looks like this:
```
----------------- ELAPSED TIME ------------------
  _Z2anIP1pE1iT_: 0.0 s
  _ZN1hIPFivEEC1EiS1_: 0.1 s
  _ZN1lC2IZ1avE3$_0EET_: 0.0 s
  _Z2auIP1pEv1oT_: 0.3 s
  _ZN1hIPFivEEC2EiS1_: 1.3 s 
  _ZN1FIP1pE2apES1_: 0.0 s
  _ZN1lC1IZ1avE3$_0EET_: 0.0 s
  _Z1av: 0.1 s
  <TOTAL>: 1.8 s
```

**Pull request merged**